File: templates/python/microgrid_market_before_after.py
"""microgrid_market_before_after: 微电网与市场前后斜率对比（slope 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='microgrid and market analysis: before-after slope'):
    return make_template_figure('slope', seed=3820, title=title, domain='microgrid and market analysis', topic='before-after slope')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/microgrid_market_before_after.m
function fig = microgrid_market_before_after()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('slope', 3820, 'microgrid and market analysis: before-after slope', 'microgrid and market analysis', 'before-after slope');
end
